### ISSUE TITLE
global_step/sec

### ISSUE THREAD

**Issue by SmallMunich** [HiKapok/SSD.TensorFlow#57]
Hi@HiKapok!
你好，我之前训练时候Linux在训练迭代过程中基本是每10步更新一个loss，
loss = 12.32 step = 50
loss = 12.13 step = 60 等

近期，我添加新的数据重复进行训练，出现许多
tensorflow:global_step/sec: 3.56742
lr = xxx   ce = xxx  loc=xxx  loss=xxx  l2=xxx  acc=xxx
重复出现

请问这样正常吗？  我感觉step = 20000+ 模型没有保存出来，不知道是什么原因
